db/version_set.cc

```diff
@@ -592,8 +615,8 @@ void LevelIterator::SeekForPrev(const Slice& target) {
   InitFileIterator(new_file_index);
   if (file_iter_.iter() != nullptr) {
     file_iter_.SeekForPrev(target);
-    SkipEmptyFileBackward();
   }
+  SkipEmptyFileBackward();
```

Comment:
I wonder if it's related to #4138

Reply:
Possibly. I just noticed this in passing.